Makefile.am

```diff
@@ -20,9 +20,6 @@ dist_noinst_DATA = logrotate.spec
 
 EXTRA_DIST = ChangeLog.md README.md autogen.sh examples
 
-# deprecated hard-wired Makefile, will be removed eventually
-EXTRA_DIST += Makefile.legacy
-
```

Comment:
@kdudka Makefile.legacy is removed in 79301953902be7ea4c49544e6e00837d0c64b72c

@glensc rpmbuild issue is fixed in fbeef99f851f0697cd806945a546835305ffabbd and I think it worked: https://travis-ci.org/logrotate/logrotate/builds/198295746

@glensc typo in 7eb58071d5322e0e74b41b797a0fa1724dfc16e9 is fixed.

Both: thank you for reviews.